wiki/tools/Qtcreator-ros.md

```diff
@@ -0,0 +1,306 @@
+---
+# Jekyll 'Front Matter' goes here. Most are set by default, and should NOT be
+# overwritten except in special circumstances. 
+# You should set the date the article was last updated like this:
+date: 2021-03-15 # YYYY-MM-DD
+# This will be displayed at the bottom of the article
+# You should set the article's title:
+title: Qtcreator User-Interface with ROS
+# The 'title' is automatically displayed at the top of the page
+# and used in other parts of the site.
+---
+[Qtcreator](https://doc.qt.io/qtcreator/index.html) is a popular user interface integrated development environment (IDE) that can interface easily with ROS. It supports multiple platforms, such as x86 desktop, arm-based embedded devices, and mobile platforms such as Android and iOS. However, this tutorial will focus on using this tool to create a user interface with ROS integrated in it.
```

Comment:
There shouldn't be any markdown in the first 100 words -- it gets displayed verbatim on the summary page.

Reply:
Hi, I am not familiar with the term. Can you explain what do you mean by excluding markdown from the first 100 words? Thanks

Reply:
This is document is in Markdown format -- note the `.md` in the file name. `[Qtcreator](https://doc.qt.io/qtcreator/index.html)` is the "markdown" formatting that I'm referring to. Jekyll (which turns this markdown document into HTML) will display this entire paragraph in the raw formatting, and not how you'd expect.

I think it's good to include a link upfront, but it should be either (a) after the first 100 words or (b) after the first line break. Again, this is only because of a limitation in Jekyll.

Reply:
Updated with b1f8624a166c5019a196e3f8f8b05d2e97a4f2eb